changelog/fix_do_not_accept_abbreviated_options_for_cli.md

```diff
@@ -0,0 +1 @@
+* [#12280](https://github.com/rubocop/rubocop/issues/12280): Do not accept abbreviated options for cli. ([@fatkodima][])
```

Comment:
```suggestion
* [#12280](https://github.com/rubocop/rubocop/issues/12280): **(Breaking)** Do not accept abbreviated options for cli. ([@fatkodima][])
```

If this change is made, it seems prudent to explicitly note `**(Breaking)**` in the changelog entry.